src/dapp/CHANGELOG.md

```diff
@@ -4,9 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## unreleased
+
+- A new type of test has been added invariant testing, 
+which runs a test function against an arbitrary contract state. More information in the [README](./README.md###invariant-testing).
```

Comment:
The expression "an arbitrary contract state" is used multiple times when describing this feature. To an unfamiliar reader, this could sound like "a specific arbitrary contract state", i.e. one that you get to specify, instead of an abstract/generic one. Maybe consider rephrasing (here and in other places) to replace arbitrary with "arbitrary symbolic" or just "generic"/"symbolic"? Also changing it to plural ("arbitrary contract states") could make this more clear.

Reply:
it's not symbolic in this iteration, this is just a fuzzed thing. Maybe `random contract state` would be better

Reply:
Oh right, in that case yes adding "random" would help make this clear since I didn't understand that this was the case. 😅